README.md

```diff
@@ -83,6 +83,74 @@ To create a new version of this package, follow these steps:
 
 [^github-activity]: If you wish, use [`github-activity` to generate a Changelog](https://github.com/choldgraf/github-activity), e.g. `github-activity conda-incubator/conda-store-ui --since 2023.9.1 --until 2023.10.1 --auth <GH personal access token>`
 
+### Running Tests
+
+This repo contains two types of tests: Jest and Playwright
+
+#### Jest tests
+
+To run the Jest tests, run the following command:
+
+```sh
+yarn test
+```
+
+#### Playwright tests
+
+Steps to install and set up:
+
+1. Create Conda environment
+   ```sh
+   conda env create -f environment.yml
```

Comment:
Oh good catch. Is there a reason we've chosen this name rather than `environment.yml`? If not, do you think I should change the file name. If so, I'll change it in this PR.

Reply:
Actually I'm just going to commit your suggestions and open a new PR if we decide to change it. Thank you for the review!